### PR TITLE
Add container helpers

### PR DESCRIPTION
**Pull request by rcooke-warwick**

### CODE REVIEW

File: core/Dockerfile.template [hunk 1]
```diff
@@ -19,9 +19,10 @@ WORKDIR /usr/app
 
 COPY --from=npm-install /tmp/node ./
 
-RUN wget https://github.com/balena-io/balena-cli/releases/download/v11.7.0/balena-cli-v11.7.0-linux-x64-standalone.zip && unzip balena-cli-v11.7.0-linux-x64-standalone.zip
+#RUN wget https://github.com/balena-io/balena-cli/releases/download/v11.7.0/balena-cli-v11.7.0-linux-x64-standalone.zip && unzip balena-cli-v11.7.0-linux-x64-standalone.zip
```
Comment (Bucknalla) [inline core/Dockerfile.template:22]: If these are commented out now, do we still need them in here?

File: core/Dockerfile.template [hunk 2]
```diff
@@ -19,9 +19,10 @@ WORKDIR /usr/app
...
 # Give global access to balena-cli
-RUN ln -sf $(pwd)/balena-cli/balena /usr/local/bin/balena
+#RUN ln -sf $(pwd)/balena-cli/balena /usr/local/bin/balena
```
Comment (Bucknalla) [inline core/Dockerfile.template:24]: Same as previous comment about commented out code

**Review by Bucknalla — CHANGES_REQUESTED** [review]
Docs updates/improvements are great! Really improves the onboarding and explains leviathan in coherent manner. Just a couple comments + probably worth squashing to tidy up what's docs vs code changes.

**Comment by rcooke-warwick** [comment]
@vipulgupta2048 tidied this up, I think we should be able to merge

**Review by vipulgupta2048 — COMMENTED** [review]
LGTM

**Review by Bucknalla — APPROVED** [review]
LGTM

**Pull request merged**